source/_integrations/vesync.markdown

```diff
@@ -46,27 +46,7 @@ This integration supports devices controllable by the VeSync App.  The following
 
 - LEVOIT Smart Wifi Air Purifier (LV-PUR131S)
 
-## Configuration
-
-To use this integration, all devices must be registered with the VeSync App. Once registration is complete, you can add the VeSync integration by adding the VeSync integration in the configuration section of the frontend and entering your username and password.  You can also use the traditional configuration method by adding the following to your `configuration.yaml` file:
```

Comment:
> To use this integration, all devices must be registered with the VeSync App.

I don't think this line should be removed, it's a prerequisite to use the integration.

Reply:
Tomorrow 😓 💤